content/docs/1.9/getting-started.md

```diff
@@ -120,7 +120,7 @@ Then navigate to `http://localhost:8080`.
 ## Migrating from Zipkin
 
 Collector service exposes Zipkin compatible REST API `/api/v1/spans` and `/api/v2/spans` for both
-JSON and thrift encoding.
+Thrift and JSON formats correspondingly.
```

Comment:
that's not accurate. The v1 endpoint accepts both JSON and Thrift, while v2 only accepts JSON (since there's no such thing as Thrift v2 in Zipkin).

https://github.com/jaegertracing/jaeger/blob/d23d18f43147431e03eedd10b82b603388964f91/cmd/collector/app/zipkin/http_handler.go#L91

Reply:
hi @d1mnewz could you please update the PR?

Reply:
Sure! Will do that right away.